src/validator_main.cpp

```diff
@@ -134,9 +134,15 @@ namespace
                 std::string errortype = (out == ebi::vcf::SUMMARY) ? "errors_summary" : "errors";
                 std::string filename = input + "." + errortype + "." + std::to_string(timestamp) + "." + filetype;
                 boost::filesystem::path file{filename};
-                if (boost::filesystem::exists(file)) {
-                    throw std::runtime_error{"Report file already exists on " + filename + ", please delete it or rename it"};
+
+                try {
+                    if (boost::filesystem::exists(file)) {
+                        throw std::runtime_error{"Report file already exists on " + filename + ", please delete it or rename it"};
+                    }
+                } catch (boost::filesystem::filesystem_error const & er) {
+                    throw std::runtime_error{"Permissions denied, can't access the location " + filename};
```

Comment:
I'm not sure about this catch and this message, the error could be another one. either test the permissions and keep the message, or remove the catch and keep only the one in odb_report.